src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepoCommandTests.cs

```diff
@@ -107,6 +109,58 @@ public async Task Happy_Path()
             mockLogger.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public async Task Idempotency_Stop_If_Target_Exists()
+        {
+            // Arrange
+            const string adoOrg = "FooOrg";
+            const string adoTeamProject = "BlahTeamProject";
+            const string adoRepo = "foo-repo";
+            const string githubOrg = "foo-gh-org";
+            const string githubRepo = "gh-repo";
+            var adoRepoUrl = $"https://dev.azure.com/{adoOrg}/{adoTeamProject}/_git/{adoRepo}";
+            var adoToken = Guid.NewGuid().ToString();
+            var githubOrgId = Guid.NewGuid().ToString();
+            var migrationSourceId = Guid.NewGuid().ToString();
+            var migrationId = Guid.NewGuid().ToString();
```

Comment:
these can be removed.

Reply:
Oops, just realized, I did not check that in. Fixed now.